### PR TITLE
Add Stompy Pro to *very* messy MuJoCo WASM

### PR DESCRIPTION
**Pull request by WT-MM**
**What does this PR do?**
Brings back mujoco component to load humanoid models and adds stompy pro

**What issues does this PR fix or reference?**
#354  - viewing mjcf files on the website

**If this PR changes the UI, include a screenshot below.**
Currently, an example is on the /test page
Screenshot 2024-09-24 at 10 53 56 AM

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -23,6 +23,24 @@
...
+        "three": "/node_modules/three/build/three.module.js"
+      }
+    }
+  </script>
+
+  <!-- Tar Libraries -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/pako/2.1.0/pako.min.js"></script>
+
+  <!-- Tensorflow.js -->
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@3.11.0/dist/tf.min.js"></script>
```
Comment (budzianowski) [inline frontend/index.html:42]: What is this needed for?

File: frontend/public/examples/main.js [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+  standardNormal,
+  toMujocoPos,
+} from "./mujocoUtils.js";
+import { DragStateManager } from "./utils/DragStateManager.js";
+
+// Load the MuJoCo Module
+const mujoco = await load_mujoco();
+
+// Set up Emscripten's Virtual File System
+var initialScene = "stompypro.xml";
```
Comment (budzianowski) [inline frontend/public/examples/main.js:21]: names should be constants in one place

File: frontend/public/examples/main.js [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+      ]);
+    }
+  }
+
+  // should re-get pawel-diff
+  async loadPPOModel() {
+    this.ppo_model = null;
+    this.getObservation = null;
+
+    // switch (this.params.scene) {
```
Comment (budzianowski) [inline frontend/public/examples/main.js:221]: Add these functionalities but remove from the code all unnecessary stuff

**Review by budzianowski — COMMENTED** [review]
please remove unused code but otherwise let's get Winston UX and shipped it

**Comment by budzianowski** [comment]
@Winston-Hsiao - we will keep it in the test page until we will have the polished version but let us know what things should be fixed from your perspective

File: frontend/index.html (hunk 1 above)
Comment (WT-MM) [reply frontend/index.html:42]: was something left over from nathan's infra for ppo

**Comment by codekansas** [comment]
cursed...

but nice job merging :) ship fast

**Pull request merged**